apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -108,14 +109,15 @@ type VSHNDBaaSNetworkSpec struct {
 	IPFilter []string `json:"ipFilter,omitempty"`
 }
 
-// VSHNDBaaSBackupSpec contains settings to control the backups of an instance.
-type VSHNDBaaSBackupSpec struct {
-	// +kubebuilder:validation:Pattern="^([0-1]?[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])$"
-	// +kubebuilder:default="21:30:00"
+type VSHNPostgreSQLBackup struct {
+	// +kubebuilder:validation:Pattern=^(\*|([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])|\*\/([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])) (\*|([0-9]|1[0-9]|2[0-3])|\*\/([0-9]|1[0-9]|2[0-3])) (\*|([1-9]|1[0-9]|2[0-9]|3[0-1])|\*\/([1-9]|1[0-9]|2[0-9]|3[0-1])) (\*|([1-9]|1[0-2])|\*\/([1-9]|1[0-2])) (\*|([0-6])|\*\/([0-6]))$
+	// +kubebuilder:default=* * * * *
+	Schedule string `json:"schedule,omitempty"`
 
-	// TimeOfDay for doing daily backups, in UTC.
-	// Format: "hh:mm:ss".
-	TimeOfDay string `json:"timeOfDay,omitempty"`
+	// +kubebuilder:validation:Pattern="^[1-9][0-9]*$"
+	// +kubebuilder:default=6
+	// +kubebuilder:validation:XIntOrString
```

Comment:
Since this is clearly an `Int` I would remove `String` validation.

Reply:
this is not a mistake if I try to leave pure int I receive:
```
component-appcat/apis/vshn/v1/dbaas_vshn_postgresql.go:119:2: must apply pattern to a `string` or `IntOrString`
```